charts/datadog/templates/_container-agent.yaml

```diff
@@ -150,6 +150,20 @@
     {{- end }}
     - name: DD_EXPVAR_PORT
       value: {{ .Values.datadog.expvarPort | quote }}
+    {{- if .Values.datadog.sbom.enabled }}
+    - name: DD_SBOM_ENABLED
+      value: "true"
+    {{- if .Values.datadog.sbom.containerImage.enabled }}
+    - name: DD_SBOM_CONTAINER_IMAGE_ENABLED
+      value: "true"
+    - name: DD_CONTAINER_IMAGE_ENABLED
+      value: "true"
+    {{- end }}
+    {{- if .Values.datadog.sbom.host.enabled }}
+    - name: DD_SBOM_HOST_ENABLED
+      value: "true"
+    {{- end }}
+    {{- end }}
```

Comment:
Since agent 7.46.0 is now release. I think we should only document the new simple solution